app/views/billing_expenses/show.html.slim

```diff
@@ -71,5 +71,55 @@
             = @billing_expense.user.profile.contact.full_name
 
 .row
+  .col-xs-7.col-xs-offset-1
+    .table-responsive
+      table.table.table-bordered
+        tbody
+          tr
+            td
+              p= t('billing_expense_pdf.aoz')
+                br=
+            td
+              p
+                = t('billing_expense_pdf.creditor')
+                br=
+            td
+              p
+                = t('billing_expense_pdf.year')
+                br= @billing_expense.created_at.year
+          tr
+            td
+              p
+                = t('billing_expense_pdf.account')
+                br= '4621'
+            td
+              p
+                = t('billing_expense_pdf.kst')
+                br= '4182'
```

Comment:
This actually puts the string inside the `<br>` tag, which is invalid HTML. Split it into two lines:

```slim
br
'4182
```